qt-api/abstractsensor_i.cpp

```diff
@@ -139,7 +139,11 @@ QDBusReply<void> AbstractSensorChannelInterface::start(int sessionId)
     connect(pimpl_->m_socketReader.socket(), SIGNAL(readyRead()), this, SLOT(dataReceived()));
 
     QList<QVariant> argumentList;
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    argumentList << QVariant::fromValue(sessionId);
+#else
     argumentList << qVariantFromValue(sessionId);
+#endif
```

Comment:
Looks like qt5 documentation already declares qVariantFromValue() as "obsolete, do not use" and suggests using
QVariant::fromValue() instead of it -> I'd suggest doing that unconditionally as it would avoid a lot of ifdef noise.

Or is there some known reason to still use qVariantFromValue()?

Reply:
For me is difficult to identify qt 5.6 compatibility issues so I have added conditional compiling. In this case it seems QVariant::fromValue was there already before so it should be safe to replace it. I am not aware of any other reasons to keep it.

Reply:
I did a quick smoketest: build qt5 packages, install on device, run csd sensor tests - did not spot any problems.

Given that the rationale for qVariantFromValue() to exist in the 1st place is given as "workaround for MSVC 6 which did not support member template functions" I think it is better to assume that it is not going to be needed anymore.